README.md

```diff
@@ -1,99 +1,104 @@
 # LBS-server
 
 # Installation Guide
+
 ## Build and Run
-### Step 1. Clone the github repo to your prefered directory using
-```
-git clone https://github.com/Suwi-inc/LBS-server.git
-``` 
-### Step 2. Inside the repository directory install the project dependencies
+
+### Step 1. Clone the github repo to your preferred directory
 ```sh
-poetry install
+git clone https://github.com/Suwi-inc/LBS-server.git
+cd LBS-server
 ```
-### Step 3: Create .env
 
-The project depends on some config that it expects to be provided from a .env file at the `LBS` directory. 
+### Step 2. Install Docker if it is not installed
+https://docs.docker.com/engine/install/
+
+### Step 3: Create .env file
+
+The project depends on some config that it expects to be provided from an .env file at the `LBS` directory.
```

Comment:
Is this config equal to the config.py file mentioned in the file structure?

Reply:
I guess, yes